parsing.js

```diff
@@ -128,7 +140,35 @@ function getVerseIndices(document) {
 }
 
 /**
- * A function that focuses on parsing the linguistics of a verse
+ *  Fetch the indices of each verse within the body of a document.
+ *  This specifically applies to the green boxes where the indices are located in the new notes.
+ *
+ *  @param {*} document
+ *  @returns an array of key value pairs (chapter:verse): index
+ */
+
+// ex: 85:1
+function getVerseIndicesTableFormat(document) {
+    let new_format;
+    let content = document.body.content;
+    let verses = {};
+    let verse;
+    content.forEach((line, line_index) => {
+        // checks if the current line contains a green table
+        // for context, the green tables are the ones that contain
```

Comment:
Make sure you are creating the template (in google docs) for note creators to base further work on. The template will essentially house all of the formatting rules with examples. This is important as it will act as a source of truth when formatting new notes (until the admin portion of the MVP app is complete).

Later we will use this consistent formatting when auto-generating notes from the data we have stored in the database.

Reply:
See for example, surat al kahf in the notes folder.